docs/docs/protocol/link-audius-account-to-protocol-dashboard.mdx

```diff
@@ -0,0 +1,87 @@
+---
+id: link-audius-account-to-protocol-dashboard
+title: Link Audius Account to Protocol Dashboard
+pagination_label: Link Audius Account to Protocol Dashboard
+sidebar_label: Link Audius Account to Protocol Dashboard
+description: Audius Protocol Documentation
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl'
+
+> Help other users identify you by connecting your [Audius](https://audius.co/) account.
+>
+> Once you've linked your Audius account, your Profile Picture and Display Name will be visible to
+> users throughout the protocol dashboard.
+
+## Connect with MetaMask
+
+To get started, open the [Audius Protocol Dashboard](https://dashboard.audius.org/) in a web browser
+with the MetaMask extension installed and click the "Connect MetaMask" button in the upper right
+hand corner of the Dashboard.
+
+<figure className="screensnippet-wrapper">
+  <img src={useBaseUrl('/img/dashboard-account/connect-metamask-button.png')} />
+  <figcaption>Connect MetaMask button.</figcaption>
+</figure>
+
+A MetaMask window will open and if there are multiple accounts associated to the MetaMask wallet, an
+account picker will be shown.
+
+Be sure to select the Wallet and Account used when registering services or making delegations to the
+Audius Protocol.
+
+<figure className="screensnippet-wrapper">
+  <img src={useBaseUrl('/img/dashboard-account/metamask-account-picker.png')} />
+  <figcaption>MetaMask account picker</figcaption>
+</figure>
+
+By default, a Gravatar style icon will be used to represent the wallet across the Dashboard.
+
+<figure className="screensnippet-wrapper">
+  <img src={useBaseUrl('/img/dashboard-account/metamask-linked.png')} />
+  <figcaption>
+    Account icon when a MetaMask account is connected to the Protocol Dashboard.
+  </figcaption>
+</figure>
+
+## Connect to Audius Profile
+
+To connect your Audius profile to the Dashboard, click the "Connect Audius Profile" button and then
+"Connect Profile".
+
+<figure className="screensnippet-wrapper">
+  <img src={useBaseUrl('/img/dashboard-account/connect-audius-profile-button.png')} />
+</figure>
+
+<figure className="screensnippet-wrapper">
+  <img src={useBaseUrl('/img/dashboard-account/connect-profile-modal.png')} />
+  <figcaption>
+    Review and confirm the next step by clicking "Connect Profile" in the modal.
+  </figcaption>
+</figure>
+
+A popup form will appear, click the "Authorize App" button to connect the accounts.
+
+If you are already signed in to an Audius account it will be chosen as the default. If you would
+like to use a different Audius account, be sure to sign out and sign in with the correct account
+before clicking "Authorize App".
+
+<figure className="screensnippet-wrapper">
+  <img src={useBaseUrl('/img/dashboard-account/authorize-app.png')} />
+</figure>
+
+Finally, MetaMask will present a signature request to confirm the account connection. Sign this
+transaction to complete the link.
```

Comment:
Nit, it's not actually a transaction. Technically it's just a message (so there is no need to pay gas, etc.)